src/runtimes/docker/clients/DockerClientBase/DockerClientBase.ts

```diff
@@ -651,6 +654,42 @@ export abstract class DockerClientBase extends ConfigurableClient implements ICo
 
     //#endregion
 
+    //#region ImagesGenerateSbom Command
+
+    /**
+     * Generate the command line arguments for invoking a pull image command on
+     * a Docker-like client

```

Comment:
Update the pull image references in the comments

Reply:
Good catch